templates/embeds/google-drive-file.php

```diff
@@ -9,10 +9,18 @@
 	// prevent direct access to this file.
 	exit;
 }
-?>
-<div style="border: 1px solid #000; text-align: center;">
-	<a href="<?php echo esc_url( $drive_file_url ); ?>" title="<?php esc_attr_e( 'Open the Shared Document', 'rt-google-embeds' ); ?>" target="_blank" rel="noopener noreferrer" style="color: #cd2653;">
-		<?php esc_html_e( 'Open Shared Document.', 'rt-google-embeds' ); ?>
-	</a>
-	<img src="<?php echo esc_url( $thumbnail_url ); ?>" alt="<?php esc_attr_e( 'Shared Document Preview', 'rt-google-embeds' ); ?>" style="border: 1px solid #eee; margin: 15px auto; display: block;" />
-</div>
+if ( file_exists( $cached_file ) ) : ?>
```

Comment:
This is not supposed to be done here. Do this before calling `render_embed` function and pass only 1 URL here. That URL will be used here directly.